package/gluon-hoodselector/files/usr/lib/micron.d/hoodselector

```diff
@@ -0,0 +1 @@
+*/2 * * * * /usr/sbin/hoodselector
```

Comment:
let's not start the lua interpreter if the lockfile exists.-

Reply:
There is no other way since nixio was removed.

Reply:
cron could do it like this:
*/2 * * * * [[ ! -f /var/lock/hoodselector.lock ]] && /usr/sbin/hoodselector

this approach means that inside the hoodselector os.exit() should be wrapped by a function that removes the lock-file.
It would save us from running a lua process in an edge case where the hoodselector is still running. It would still be executed in the majority of cases where nothing changed which is the bigger burden anyway. Since I do not know how to change that aspect which weighs heavier I guess the current implementation is fine with me.

Reply:
Deleting lock-files is dangerous; if not implemented correctly, it may break locking. My recommendation is never to delete lock files.

Reply:
Every two minutes sounds like we're chasing a moving target. Is that really the case?

Reply:
@mweinelt Could be possible.